### ISSUE TITLE
Specifying an action at the end of the gather() method

### ISSUE THREAD

**Issue by steveanton** [w3c/webrtc-ice#8]
Currently the `gather()` specification just includes validation and does not specify an action connected to the ICE agent.

Should we add something like: instruct the ICE agent to start gathering candidates?

Peter suggested pointing to section 5.1.1. from https://datatracker.ietf.org/doc/draft-ietf-ice-rfc5245bis/?include_text=1

**Comment by aboba** [comment]
Yes.  It would not make much sense if `gather()` just did validation but did not cause the ICE agent to start gathering. :)
